server/src/core/server/models/comment/comment.ts

```diff
@@ -110,6 +110,11 @@ export interface Comment extends TenantResource {
    */
   status: GQLCOMMENT_STATUS;
 
+  /**
+   * initialStatus is the initial Comment Status.
+   */
+  initialStatus: GQLCOMMENT_STATUS;
```

Comment:
Thoughts on storing/exposing this in the comment revisions? We follow a history based structure everywhere else in the server, might be cool to have the status per-revision?

Reply:
Yeah, I like this idea so that we have access to it for reach revision this way. Updated to then use the first revision status to populate `initialStatus` for comments then.

Reply:
amazing, thank you!